src/adapters/surveyAdapter.js

```diff
@@ -16,22 +16,6 @@ class SurveyAdapter {
       }
     });
   };
-
-  static get = (surveyId, token) => {
```

Comment:
@rafayet-monon Is this because we didn't rebase before merge? 🤔

Reply:
Yeah, Something like that. I'm not so sure either when it got there. I had to rebase these branches almost 7-8 times. Something might have happened in between. 😞